frontend/src/pages/SubmitProperty.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+import { IProperty, IParcelDetail } from 'actions/parcelsActions';
+import { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import * as reducerTypes from 'constants/reducerTypes';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { useKeycloak } from '@react-keycloak/web';
+import { mountToJson } from 'enzyme-to-json';
+import SubmitProperty from './SubmitProperty';
+
+jest.mock('@react-keycloak/web');
+Enzyme.configure({ adapter: new Adapter() });
+(useKeycloak as jest.Mock).mockReturnValue({
+  keycloak: {
+    userInfo: {
+      agencies: ['1'],
+    },
+    subject: 'test',
+  },
+});
+const mockStore = configureMockStore([thunk]);
+
+// This will spoof the active parcel (the one that will populate the popup details)
+const mockDetails: IParcelDetail = {
+  propertyTypeId: 0,
+  parcelDetail: {
+    id: 1,
+    pid: '000-000-000',
+    pin: '',
+    statusId: 0,
+    classificationId: 0,
+    zoning: '',
+    zoningPotential: '',
+    agencyId: 0,
+    latitude: 48,
+    longitude: 123,
+    propertyStatus: 'active',
+    classification: 'Core Operational',
+    description: 'test',
+    isSensitive: false,
+    evaluations: [
+      {
+        date: '2019',
+        key: '',
```

Comment:
Did you want to test an empty string?  I thought I created enums that you could use.